ui/pages/Search/components/VariantSearchResults.jsx

```diff
@@ -37,16 +37,19 @@ const FIELDS = [
   VARIANT_SORT_FIELD_NO_FAMILY_SORT,
 ]
 
-
 const BaseVariantSearchResults = ({
   match, searchedVariants, variantSearchDisplay, searchedVariantExportConfig, onSubmit, load, unload, loading, errorMessage, totalVariantsCount,
 }) => {
+  const scrollToTop = () => window.scrollTo(0, 0)
```

Comment:
you should actually define this as a constant outside of the  component, as it does not require any of the component props and theres therefore no reason to have it re-created on render.